fission/function.go

```diff
@@ -373,6 +379,37 @@ func fnUpdate(c *cli.Context) error {
 	entrypoint := c.String("entrypoint")
 	buildcmd := c.String("buildcmd")
 	force := c.Bool("force")
+	secretName := c.String("secret")
```

Comment:
Did some mis-merge happen here? Why is this change in this PR?

Reply:
So I could not find the code for updating secrets and configmaps, so added this.

Reply:
Let me move this change to another PR.